Server/mods/deathmatch/logic/CMainConfig.cpp

```diff
@@ -133,6 +133,9 @@ bool CMainConfig::Load()
         return false;
     }
 
+    // Strip spaces from beginning and end of server name
+    m_strServerName = SString(m_strServerName).TrimStart(" ").TrimEnd(" ");
+
```

Comment:
It's ok in this context, but TrimStart and TrimEnd suck

Reply:
Ok but why do they suck?